modules/cli/src/test/java/org/apache/ignite/internal/cli/core/repl/completer/DynamicCompletionInsiderTest.java

```diff
@@ -31,12 +31,6 @@ class DynamicCompletionInsiderTest {
 
     private static Stream<Arguments> typedWordsSource() {
         return Stream.of(
-                // connect command needs a parameter
-                Arguments.of(new String[]{"connect"}, false),
```

Comment:
I think these test cases should not be removed but replaced with another command.